plugin-gradle/src/test/java/com/diffplug/gradle/spotless/JsonExtensionTest.java

```diff
@@ -156,4 +156,44 @@ void jacksonFormattingWithSortingByKeys() throws IOException {
 		gradleRunner().withArguments("spotlessApply").build();
 		assertFile("src/main/resources/example.json").sameAsResource("json/sortByKeysAfter_Jackson.json");
 	}
+
+	@Test
+	void applyJsonPatchReplaceString() throws IOException {
+		setFile("build.gradle").toLines(
+				"plugins {",
+				"    id 'java'",
+				"    id 'com.diffplug.spotless'",
+				"}",
+				"repositories { mavenCentral() }",
+				"spotless {",
+				"    json {",
+				"        target 'src/**/*.json'",
+				"        applyJsonPatch([[op: 'replace', path: '/abc', value: 'ghi']])",
+				"        gson()",
```

Comment:
The tests post-process this step's output with another formatter.

zjsonpatch uses Jackson under the hood but I didn't want to get into all the duplication of configuring its pretty printing like it's done in the Jackson steps.

I guess most users will want to apply formatting steps after patching anyway!